src/styles/blocks/modifiers.scss

```diff
@@ -0,0 +1,17 @@
+.stars--4 .stars__item:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__button--styles {
+  color: white;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 40px;
+
+  &:hover {
+    background-color: #fff;
+    color: rgb(0, 172, 220);
+    border: 1px solid rgb(0, 172, 220);
+  }
+}
```

Comment:
The `card__button--styles` class seems to be a modifier for `card__button`. However, it's not clear what `--styles` means. Modifiers should describe the state or a specific characteristic of the block or element they belong to. Consider renaming this modifier to something more descriptive.